.travis.yml

```diff
@@ -14,10 +14,6 @@ matrix:
         env: TOXENV=isort,lint
       - python: "3.7"
         env: TOXENV=docs
-      - python: "2.7"
-        env: TOXENV=py27-plone-5.2
-      - python: "2.7"
-        env: TOXENV=py27-plone-5.2-archetypes
       - python: "3.6"
         env: TOXENV=py36-plone-5.2
```

Comment:
Probably it would be good to move this to GHA once we have a Plone 6 alpha released